pkg/planner/core/testdata/plan_suite_unexported_out.json

```diff
@@ -73,16 +73,16 @@
       "Join{DataScan(t1)->DataScan(t2)}(test.t.a,test.t.b)(test.t.d,test.t.d)->Projection",
       "Join{DataScan(ta)->DataScan(tb)}(test.t.d,test.t.d)->Projection",
       "Join{DataScan(ta)->DataScan(tb)}(test.t.d,test.t.d)->Projection",
+      "Join{DataScan(ta)->DataScan(tb)}(test.t.d,test.t.d)->Sel([eq(test.t.a, 0)])->Projection",
```

Comment:
Why this output changed a lot ? I saw you didn't change any case of this in.json ?

Reply:
@elsa0520 : actually some tests are missing flagConvertOuterToInnerJoin whcih cause some joins not converted and different push down. Will fix the tests by adding flagConvertOuterToInnerJoin to the plan tests.